iOS/FVM/FVMTests/MaterialProcessorTests.swift

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import XCTest
+import SceneKit
+@testable import FVM
+
+public class MaterialProcessorTests: XCTestCase {
+    
+    func test() {
+        let processor = MaterialProcessor()
```

Comment:
Should we add AAA comments? Like: //Arrange //Act //Assert? @lapwingg @ghawliczek

Reply:
Or // given // when // then

Which one do you prefer more?

Reply:
None. It should come out from the code. Just empty lines between each part.